packer/linux/scripts/install-buildkite-agent.sh

```diff
@@ -12,14 +20,17 @@ sudo useradd --base-dir /var/lib --uid 2000 buildkite-agent
 sudo usermod -a -G docker buildkite-agent
 
 echo "Downloading buildkite-agent v${AGENT_VERSION} stable..."
-sudo curl -Lsf -o /usr/bin/buildkite-agent-stable \
-  "https://download.buildkite.com/agent/stable/${AGENT_VERSION}/buildkite-agent-linux-amd64"
-sudo chmod +x /usr/bin/buildkite-agent-stable
+mkdir -p buildkite-agent
+curl -Lsf -o buildkite-agent-linux.tar.gz \
+  "https://github.com/buildkite/agent/releases/download/v${AGENT_VERSION}/buildkite-agent-linux-${ARCH}-${AGENT_VERSION}.tar.gz"
+tar -C buildkite-agent -xvzf buildkite-agent-linux.tar.gz
+sudo cp buildkite-agent/buildkite-agent /usr/bin/buildkite-agent-stable
+rm -fr buildkite-agent buildkite-agent-linux.tar.gz
```

Comment:
Hmm, I just noticed this changes from downloading the binary to downloading and unpacking a tarball. @galak, was that because this URL didn't work at the time? https://download.buildkite.com/agent/stable/3.25.0/buildkite-agent-linux-arm64